back-nest/src/main.ts

```diff
@@ -19,7 +20,7 @@ async function bootstrap() {
 
   const PORT = process.env.PORT;
   await app.listen(PORT);
-  console.log(`Server is running on http://localhost:${PORT}`);
+  Logger.log(`Server is running on http://localhost:${PORT}`);
```

Comment:
I would recommend you to add logger into app.module dependencies, so that you can get it as a injected service instead of library class. Check comment in app.module file and proceed with these changes then:

```ts
  const logger = app.get(Logger);
  logger.log(`Server is running on http://localhost:${PORT}`);
```